src/components/BackgroundContainer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { string } from 'prop-types';
+
+BackgroundContainer.propTypes = {
+    img: string
+};
```

Comment:
Add types for children and filter properties

Reply:
Done